andela/mag-ah-frontend
# #159965266 Users should be able to login to Author's Haven

#### What does this PR do?
 * Implementation of login feature for front-end.

#### Description of Task to be completed?
 * The login feature allows a user to log in via front-end.

#### How should this be manually tested?
 * Clone our repository: https://github.com/andela/mag-ah-frontend.git
 * Checkout to: ft-login-159965266
 * Install requirements: `npm install`
 * Run app: `npm start`

On the header of the page :
 * Click on Sign in button
 * Provide email and password on the modal that appears

#### What are the relevant pivotal tracker stories?
 * Homepage
<img width="1095" alt="screen shot 2018-10-17 at 11 50 45" src="https://user-images.githubusercontent.com/30254491/47074394-fb23e000-d202-11e8-936d-744137fcf8e8.png">

 * Sign in modal
<img width="616" alt="screen shot 2018-10-17 at 11 52 57" src="https://user-images.githubusercontent.com/30254491/47074538-46d68980-d203-11e8-915d-8968b14d4a79.png">



[User should be able to login to the application](https://www.pivotaltracker.com/n/projects/#159965266)
